oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/RelatedSelectField/ExternalApiResultsList.jsx

```diff
@@ -28,56 +31,44 @@ export const ExternalApiResultsList = withState(
         serializeExternalApiSuggestions={serializeExternalApiSuggestions}
       >
         <Grid celled columns={3}>
-          {serializeExternalApiSuggestions(results?.data?.hits).map(
-            (record) => {
-              const title = record.text;
-              const isChecked = externalApiRecords.some(
-                (record) => record.text === title
-              );
-              const isDisabled = externalApiRecords.length >= 20 && !isChecked;
-              return multiple ? (
-                <Grid.Row width={10} key={record.value}>
-                  <Grid.Column width={2}>
-                    <Checkbox
-                      disabled={isDisabled}
-                      checked={isChecked}
-                      onChange={() => {
-                        handleExternalRecordChange(record);
-                      }}
-                    />
-                  </Grid.Column>
-                  <Grid.Column width={8}>
-                    <Header size="small" className="mt-0">
-                      {title}
-                    </Header>
-                  </Grid.Column>
-                </Grid.Row>
-              ) : (
-                <Grid.Row
-                  key={record.value}
-                  onClick={() => {
-                    handleAddingExternalApiSuggestion([record]);
-                    handleExternalRecordChange(record);
-                  }}
-                >
-                  <Grid.Column width={2}>
-                    <Radio
-                      checked={externalApiRecords[0]?.text === title}
-                      onChange={() => handleExternalRecordChange(record)}
-                    />
-                  </Grid.Column>
-                  <Grid.Column width={8}>
-                    <Header size="small" className="mt-0">
-                      {title}
-                    </Header>
-                  </Grid.Column>
-                </Grid.Row>
-              );
-            }
-          )}
+          {serializedSuggestions.map((record) => {
+            const title = record.text;
```

Comment:
`record.text` is not unique identifier of a record, we should use key or `record.id`